mkdocs.yml

```diff
@@ -133,6 +133,7 @@ nav:
           - The Search Bar: "user-guides/getting-started/search-bar.md"
           - Git as a Data Source: "user-guides/git-data-source.md"
           - GraphQL: "user-guides/graphql.md"
+          - Custom Fields: "user-guides/custom-fields.md"
```

Comment:
I'll register my usual complaint here that I think our separate `User Guides` section is not a great pattern. We already have `models/extras/customfield.md`, what's the purpose/benefit to having a new, separate, second page of documentation for custom fields, instead of adding all of this great content to the existing page?

Reply:
Agreed that this is not much of a user guide but more in depth documentation.